repos/system_upgrade/common/actors/checkipaserver/libraries/checkipaserver.py

```diff
@@ -3,7 +3,7 @@
 
 MIGRATION_GUIDE_7 = (
```

Comment:
Maybe let's use shortener for this link as well, so that it is easily testable?

Reply:
I agree with Ina. I would use the shortened URL so in case of another change, we can reflect the broken symlink without the need to modify the code.

Reply:
Alright, updated